server/kit/kitserver.go

```diff
@@ -89,27 +93,33 @@ func NewServer(svc Service) *Server {
 		propr propagation.HTTPFormat
 	)
 
-	projectID := googleProjectID()
+	projectID := observe.GoogleProjectID()
 	var svcName, svcVersion string
-	if isGAE() {
-		_, svcName, svcVersion = getGAEInfo()
+	if observe.IsGAE() {
```

Comment:
Since we have very similar logic in the new package, i wonder if we could lean on that here?

Maybe something like `observe.GetServiceInfo()` that returns the project, service and version and we call it within the `observer.IsGCPAccessible()` block?